coriolis/providers/backup_writers.py

```diff
@@ -169,6 +169,13 @@ def __init__(self, path, disk_id, compress_transfer=None):
         self._stderr = None
         self._offset = None
         self._ssh = None
+        self._sender_q = eventlet.Queue(maxsize=5)
+        self._enc_q = eventlet.Queue(maxsize=5)
+        self._sender_evt = None
+        self._encoder_evt = None
+        self._exception = None
+        self._closing = False
```

Comment:
NIT: I'd rename this to `self._closed` to better indicate it represents the state of being closed or not

Reply:
It represents a transition to closing state. When we call ```close()```, there may still be chunks in transit. The ```close()``` function waits for eventlets to finish sending chunks before it returns. However, we still need to make sure no new ```write()``` operations are allowed if ```close()``` is called.

So in short, this variable indicates that we have called ```close()``` but some operations may still be pending.